src/tokens.md

```diff
@@ -368,7 +369,10 @@ Like any literal, an integer literal may be followed (immediately,
 without any spaces) by an _integer suffix_, which forcibly sets the
 type of the literal. The integer suffix must be the name of one of the
 integral types: `u8`, `i8`, `u16`, `i16`, `u32`, `i32`, `u64`, `i64`,
-`u128`, `i128`, `usize`, or `isize`.
+`u128`, `i128`, `f32`,`f64`, `usize`, or `isize`.
+
+Note the suffix of `f32` and `f64` enforce that an integer literal can coerced
+into a float of repestive suffix type, which the lexer can be permissive of.
```

Comment:
```suggestion
Note the suffix of `f32` and `f64` ensures that the literal has a floating point type.
Even when it would otherwise be lexed as an integer.
```